src/core/bridge.ts

```diff
@@ -28,6 +36,7 @@ export const DEFAULT_CONFIG: BridgeConfig = {
 export abstract class AbstractBridge {
 
   config: BridgeConfig;
+  fetch: fetchPonyfill;
```

Comment:
is this the correct type?

Reply:
I'm not sure what the correct type is here, this seemed most correct to me, is there a better one?